### ISSUE TITLE
Add a web client for MQTT-over-Websocket

### ISSUE THREAD

**Issue by mcollina** [mcollina/mosca#87]
We can include the code in https://github.com/hivemq/hivemq-mqtt-web-client, it's apache licensed.

**Comment by roccomuso** [comment]
Old issue but, what do you meant initially with "add a web client"? you mean a client with a cool UI?

**Comment by mcollina** [comment]
Yes, something to publish and subscribe to the broker and do general debugging.

**Comment by roccomuso** [comment]
Hmm I could work on that, should it have a basic UI or we're looking for something cool to see? material design etc..

**Comment by mcollina** [comment]
If you are into polymer, you can try https://github.com/mqttjs/mqtt-elements.

Basic UI, and you can also leverage from mqttlens (https://github.com/sandro-k/MQTTLensChromeApp) by @sandro-k.

**Comment by mcollina** [comment]
Anyway, any help is very welcomed!

**Comment by mcollina** [comment]
(or something react-based is fine too!)

**Comment by sandro-k** [comment]
I am trying to find some time to update MQTTLens to the mqtt-elemets and polymer 1.0. Maybe that can help you.

**Comment by YasserTahri** [comment]
I have used mqtt for web client but i hve this error message:
failed: Error during WebSocket handshake: Unexpected response code: 200 !!!!!

**Comment by sandro-k** [comment]
@YasserTahri Please open up a new issue and explain your problem in more depth there.
